requirements.txt

```diff
@@ -5,5 +5,5 @@ requests
 beautifulsoup4
 numpy
 scipy
-git+https://github.com/msrb/nvdlib.git#egg=nvdlib
+git+https://github.com/fabric8-analytics/nvdlib.git#egg=nvdlib
```

Comment:
Don't we need to pin the version?

Reply:
Thanks, Pavel, It might be a good idea once the https://github.com/fabric8-analytics/nvdlib/pull/6 PR is merged. I'll add relevant tags for the [nvdlib](https://github.com/fabric8-analytics/nvdlib) afterwards.